modules/zstr/1.0.7/MODULE.bazel

```diff
@@ -0,0 +1,5 @@
+module(
+    name = "zstr",
+    version = "1.0.7",
+    compatibility_level = 1,
```

Comment:
you can add  `bazel_compatibility = [">=7.2.1"],  # need support for "overlay" directory`

Reply:
Done